docs/dns-client/configuration.md

```diff
@@ -38,6 +38,28 @@ The `cache` object configures caching the results of querying DNS. It has the fo
 
 The `server` object configures the handling of incoming requests. It has the following properties:
 
+  - `bind_retry`: The confguration of the retry mechanism for binding to the listen addresses. This is useful if the server is started before the network is ready and the addresses are not yet available, as on some editions of Windows when installed as a system service. It can increase the startup time, so it should be disabled if not needed.
+
+    :::note
+
+    This object is available since **v0.0.3**.
```

Comment:
@el-termikael, please remind me, do we have any sort of guidelines regarding whether bold and/or italic formatting stops at punctuation or takes over it?  That is, whether it's `**v0.0.3**.` or `**v0.0.3.**`?

Reply:
I prefer the first option where formatting stops at punctuation